### PR TITLE
Minor installer improvements

### PR DESCRIPTION
**Pull request by tobim**
* Clear the destination path of the package download so `curl` does not abort.
* Drop the check for `adduser`, because it is out of place in the installer.
* Add `/opt/tenzir/bin` to the system default path for ease of use.

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -209,6 +210,7 @@ fi
 
 # Trigger installation.
 action "Installing package into ${prefix}"
+insert_path="$sudo echo 'export PATH=\$PATH:/opt/tenzir/bin' > /etc/profile.d/tenzir.sh"
```
Comment (tobim) [inline scripts/install.sh:213]: I'm not sure how to achieve this on macOS.

Comment (mavam) [reply scripts/install.sh:213]: There's no equivalent. Shells have their specific way of doing this, e.g., via `.bash_profile`.

Comment (mavam) [reply scripts/install.sh:213]: Can we create a symlink that points from `/usr/local/bin/tenzir` to `/opt/tenzir/bin/tenzir`?

**Comment by mavam** [comment]
@tobim are you moving the `/etc/profile.d` actions to the RPM/DEB package in this PR?

**Pull request merged**